src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -130,6 +149,24 @@ public boolean matches(String database, String table) {
 		return matchesDatabase(database) && matchesTable(table);
 	}
 
+	private boolean matchesValues(Map<String, Object> data) {
+		for (Map.Entry<String, String> entry : includeColumnValues.entrySet()) {
+			String column = entry.getKey();
+
+			if (data.containsKey(column)) {
+				String columnValue = entry.getValue();
+
+				Object value = data.get(column);
+				if (value == null) return false;
```

Comment:
I could definitely see value in including NULL columns; we should special case that (if the user passes in C=null, it matches both NULL and "null".  Obviously kinda odd and non-ideal but hey it's ok for government work)

Reply:
We will add support for null value.